lib/src/components/NxSubmitMask/__tests__/NxSubmitMask.test.tsx

```diff
@@ -42,17 +41,18 @@ describe('NxSubmitMask', function() {
 
   it('passes the successMessage prop as the child of NxLoadingSpinner when the success prop is true', function() {
     const spinnerChildren = shallow(<NxSubmitMask message="foo" successMessage="bar" success />)
-        .find(NxLoadingSpinner).children();
+        .find('div.nx-submit-mask__message span').children();
```

Comment:
Selecting unclassed elements like this isn't terribly maintainable.  I'd rather there be a class added to the span.

Reply:
> Selecting unclassed elements like this isn't terribly maintainable. I'd rather there be a class added to the span.

Done: https://github.com/sonatype/sonatype-react-shared-components/pull/182/commits/7eaad8766f8001a37f8fff6f890d10ea7a8be8af